README.md

```diff
@@ -6,41 +9,40 @@
 2. [Module Description - What the module does and why it is useful](#module-description)
 3. [Setup - The basics of getting started with concat](#setup)
     * [What concat affects](#what-concat-affects)
-    * [Setup requirements](#setup-requirements)
     * [Beginning with concat](#beginning-with-concat)
 4. [Usage - Configuration options and additional functionality](#usage)
-    * [API _deprecations_](#api-deprecations)
 5. [Reference - An under-the-hood peek at what the module is doing and how](#reference)
-5. [Limitations - OS compatibility, etc.](#limitations)
-6. [Development - Guide for contributing to the module](#development)
+    * [Public Defines](#public-defines)
+    * [Parameters](#parameters)
+    * [Deprecations](#deprecations)
+6. [Limitations - OS compatibility, etc.](#limitations)
+7. [Development - Guide for contributing to the module](#development)
 
 ##Overview
 
-This module constructs files from multiple fragments in an ordered way.
+The concat module lets you construct files from multiple ordered fragments of text.
 
 ##Module Description
 
-This module lets you use many concat::fragment{} resources throughout
-your modules to construct a single file at the end.  It does this through
-a shell (or ruby) script and a temporary holding space for the fragments.
+The concat module lets you gather `concat::fragment` resources from your other modules and order them through a single `concat` resource into a coherent file. It does this through a shell (or Ruby) script and a temporary holding space for the fragments.
```

Comment:
I think this changed while you were working on the README, but it's just Ruby script now :)